### PR TITLE
Add Charon 3.0.0

### PR DESCRIPTION
**Pull request by Vindulamj**

### CODE REVIEW

**Comment by cdwijayarathna** [comment]
Have we discussed the architecture of this anywhere? Changes from 2.x.x to this one?

File: modules/charon-core/pom.xml [hunk 1]
```diff
@@ -3,14 +3,14 @@
     <parent>
         <groupId>org.wso2.charon</groupId>
         <artifactId>charon-parent</artifactId>
-        <version>2.0.7-SNAPSHOT</version>
+        <version>3.0.0</version>
```
Comment (mefarazath) [inline modules/charon-core/pom.xml]: shouldn't the version be 3.0.0-SNAPSHOT

File: modules/charon-core/pom.xml [hunk 2]
```diff
@@ -3,14 +3,14 @@
...
         <relativePath>../../pom.xml</relativePath>
     </parent>
 
     <modelVersion>4.0.0</modelVersion>
-    <artifactId>org.wso2.charon.core</artifactId>
+    <artifactId>org.wso2.charon.core.v2</artifactId>
     <packaging>bundle</packaging>
-    <name>WSO2 Charon - Core</name>
+    <name>WSO2 Charon - Core - V2</name>
```
Comment (mefarazath) [inline modules/charon-core/pom.xml]: why V2?

File: modules/charon-core/pom.xml [hunk 3]
```diff
@@ -52,18 +52,19 @@
...
-                            org.wso2.charon.core.*;version="${charon.parent.package.export.version}"
+                            org.wso2.charon.core.*;version=3.0.0
                         </Export-Package>
                         <Import-Package>
                             org.json.*,
                             org.apache.commons.logging.*; version="${commons-logging.osgi.version.range}",
-                            org.apache.axiom.om.*; version="${axiom.osgi.version.range}",
-                            javax.xml.namespace.*,
-                            javax.xml.stream.*,
+                            <!--org.apache.axiom.om.*; version="${axiom.osgi.version.range}",-->
```
Comment (mefarazath) [inline modules/charon-core/pom.xml]: Let's get rid of the commented imports if we don't need them.

**Pull request closed without merging**